src/parser/bison_parser.y

```diff
@@ -144,6 +144,7 @@
 
   std::vector<char*>* str_vec;
   std::vector<hsql::ConstraintType>* column_constraint_vec;
```

Comment:
column_constraint_vec should be removed, because it is replaced by column_constraint_set, right?

Reply:
absolutely